src/main/java/com/hubspot/jinjava/util/ChunkResolver.java

```diff
@@ -265,7 +268,7 @@ private String resolveChunk(String chunk) {
       String resolvedChunk;
       Object val = interpreter.resolveELExpression(chunk, token.getLineNumber());
       if (val == null) {
-        return JINJAVA_NULL;
+        resolvedChunk = JINJAVA_NULL;
```

Comment:
Fixes minor whitespace issue that was found from the test I added, where the space before a null value would be removed. Short-circuiting here prevented the space from being preserved, so this makes sure to preserve the space if it exists